cukinia/common_security_tests.d/sudo.conf.j2

```diff
@@ -44,6 +44,17 @@ CMD_WITHOUT_ARGS=$(
 as "SEAPATH-00154 - sudo commands always specify arguments" \
     cukinia_test -n $CMD_WITHOUT_ARGS
 
+USER_ACCESS_ALL=$(
+    grep -h "ALL.*=.*ALL" $SUDOERS_FILES |
+    grep -Ev 'root|sudo' |
+    grep -v '{{ admin_user }}'
+)
+# The first line matches all the sudoers lines who let a user run ALL.
+# The second line remove sudo and root who must be able to target ALL.
+# The third line is an exception. Currently, admin user can run ALL.
+as "SEAPATH-00154 - no user can run all commands as root" \
```

Comment:
you are using "SEAPATH-00154" as test id. This id is already used just above, is this normal ?

Reply:
Yes, because these two cukinia tests are two parts of the same test.
That is something that we have already done in SEAPATH. And that works well with the test report generation